.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -463,7 +463,10 @@ jobs:
           # interruption in such flows.
           cilium --context ${{ env.contextName1 }} connectivity test \
             --multi-cluster=${{ env.contextName2 }} --hubble=false \
-            --include-conn-disrupt-test --include-conn-disrupt-test-ns-traffic \
+            --include-unsafe-tests \
+            --include-conn-disrupt-test \
+            --include-conn-disrupt-test-ns-traffic \
+            --include-conn-disrupt-test-egw \
```

Comment:
EGW is not enabled in this test (and doesn't work cross-cluster) so this is not strictly necessary here.